javascript/schema/src/codebook/ValidationResult.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { ColumnError } from './ColumnError.js'
+
+export interface ValidationResult {
+	columnErrors?: ColumnError[]
```

Comment:
I would call this `errors` for simplicity (and you could reuse it elsewhere that we do validation)